src/api/java/appeng/core/crafting/api/definitions/ICraftingTileDefinitions.java

```diff
@@ -11,4 +10,21 @@
 public interface ICraftingTileDefinitions extends IDefinitions<Class<? extends TileEntity>, ITileDefinition<? extends TileEntity, Class<? extends TileEntity>>>
 {
 
+    default ITileDefinition tileCraftingMonitor(){
+        return get( "crafting_monitor ");
+    }
+
+    default ITileDefinition tileCraftingStorage(){
+        return get( "crafting_storage" );
+    }
+
+    default ITileDefinition tileCraftingUnit(){
+        return get( "crafting_unit" );
+    }
+
+    default ITileDefinition tileMolecularAssembler(){
```

Comment:
We are already in tile definitions, so adding tile/block/item prefix is completely unnecessary.